synapse/events/__init__.py

```diff
@@ -339,8 +343,8 @@ def event_id(self) -> str:
         raise NotImplementedError()
 
     @property
-    def membership(self) -> str:
-        return self.content["membership"]
+    def membership(self) -> Optional[str]:
+        return self.content.get("membership")
```

Comment:
Why this change? I'm struggling to think through whether this is safe enough to do or not.

Reply:
Thanks for noticing! It was relevant for the [old version of is_notifiable()](https://github.com/Famedly/synapse/blob/eaf3ab1436c3360f82b9f8b18c14aacfdea8818c/synapse/events/__init__.py#L349), but after requested changes, it's not necessary anymore. It should be absolutely safe, but I will revert it.